src/Command/PullRequest/PullRequestCloseCommand.php

```diff
@@ -47,11 +48,16 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $prNumber = $input->getArgument('pr_number');
-        $closingComment = $input->getOption('message');
-
         $adapter = $this->getAdapter();
+        $prNumber = $input->getArgument('pr_number');
+        $pr = $adapter->getPullRequest($prNumber);
+        $authenticatedUser = $this->getParameter($input, 'authentication')['username'];
+        $removeSourceBranch = $input->getOption('remove-source-branch');
```

Comment:
Indeed, value should not be required. That's the common pattern for bool options.